### DISCUSSION TITLE
Generating/Decompiling OpenType feature from binary back to Adobe FEA?

### DISCUSSION THREAD

**Discussion by NightFurySL2001** [fonttools/fonttools#3151] [Q&A]
Is it possible with FontTools to generate/decompile OpenType features in binary font file into Adobe FEA? It might be handy for generating editable source files from Adobe FEA (e.g. UFO). 

Currently I only found ft2fea on https://github.com/graphicore/extractFea which doesn't work now with lots of bugs and unimplemented features (e.g. not supporting multiple lookup subtables in `formatLookupMarkToBase`).

The other way is to use FontForge....

**Comment by moyogo** [comment]
Have you tried https://github.com/simoncozens/fontFeatures?
